JASP-Desktop/gui/preferencesmodel.h

```diff
@@ -38,7 +38,8 @@ class PreferencesModel : public QObject
 	Q_PROPERTY(bool			safeGraphics			READ safeGraphics				WRITE setSafeGraphics				NOTIFY safeGraphicsChanged				)
 	Q_PROPERTY(QString		cranRepoURL				READ cranRepoURL				WRITE setCranRepoURL				NOTIFY cranRepoURLChanged				)
 	Q_PROPERTY(int			plotPPI					READ plotPPI														NOTIFY plotPPIPropChanged				)
-	Q_PROPERTY(QFont		defaultFont				READ defaultFont				WRITE setDefaultFont				NOTIFY defaultFontChanged				)
+	Q_PROPERTY(QString		interfaceFont			READ interfaceFont				WRITE setInterfaceFont				NOTIFY interfaceFontChanged				)
```

Comment:
Also a problem, by changing QFont to QString we will no longer be able to use non-system fonts.
Like the open source sansserif font included with JASP.
Which it would be great to use!

Because it contains all mathematical symbols for sure.

Reply:
But we do not load FreeSans yet. And I think if we add it in the application, we can call it by its name (we will have to anyway, by selecting it in the Dropdown).

Reply:
Ooh yeah, ok. 

But I still think QFont is the way to go.